### PR TITLE
The new load package reads and does some validation of deployment con…

### PR DESCRIPTION
**Pull request by bormanp**
…figs.

The load package has no direct knowledge of the Deployment or DeploymentConfig structures.

New controllers can be added without needing to modify the kne deployment code.

The load package can be used independently of the KNE CLI to validate deployment configurations.

Currently it does not validate that the images are present as this must be done from within the KNE cluster.

### CODE REVIEW

**Comment by bormanp** [comment]
I am still verifying that KNE deploys correctly.  Creating the PR now to give a change to look at what I have done.

**Comment by bormanp** [comment]
The golangci-lint failure is the known failure mentioned before.

File: load/deploy_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
+package load_test
```
Comment (alexmasi) [inline load/deploy_test.go:1]: will this package naming overlap when load topology is added?

**Review by alexmasi — APPROVED** [review]

File: load/deploy_test.go (hunk 1 above)
Comment (bormanp) [reply load/deploy_test.go:1]: No, I plan to add the topology code into this same package.

**Pull request merged**